simmenu.h

```diff
@@ -82,6 +82,7 @@ enum {
 	TOOL_GENERATE_SCRIPT,
 	TOOL_REMOVE_HALT,
 	TOOL_EXTINGUISH_WAITING_GOODS,
+	TOOL_CHANGE_CITY_OF_CITYBUILDING,
```

Comment:
(naming) `TOOL_CHANGE_CITY_OF_CITY_BUILDING` or `TOOL_CHANGE_CITY_OF_BUILDING`?

Reply:
`TOOL_CHANGE_CITY_OF_CITY_BUILDING` is probably the best option since "City of Building" doesn't properly show what the tool does since there are also buildings that aren't linked to any city like the attractions etc.